theorems/T000232.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000232
+if:
+  and:
+  - P000046: true
+  - P000097: true
+then:
+  P000050: true
+refs:
+  - mathse: 3824535
```

Comment:
I think https://math.stackexchange.com/questions/3824078/a-subspace-of-mathbbr-is-zero-dimensional-iff-it-is-totally-disconnected/3824535#comment7886688_3824078 suggests "embeddable in R" can be generalized to a "GO-space" (subspace of any LOTS). perhaps a future contribution.

Reply:
Absolutely.  That was my plan after this: add GO-space as a property. Many theorems we already have for LOTS can be generalized for those.